core/src/test/resources/Message/testStringWithXmlDeclaration.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<root><sub>abc&amp;&lt;&gt;</sub><sub><![CDATA[<a>a&amp;b</a>]]></sub><data attr="één €">één €</data></root>
```

Comment:
Misschien wel zo handig hier utf-8 characters in te zetten? Zoals de inhoud van `/Util/MessageUtils/utf8-with-bom.txt`.

Reply:
Ik begrijp je nog niet helemaal. Bedoel je in de filenaam?

De characters é en € zijn al UTF-8 encoded, die zitten niet in de standaard 7-bit ANSII characterset.